game.py

```diff
@@ -611,7 +611,7 @@ def end_turn(self) -> bool:
                 if self.until_night == 0:
                     self.board.overlay.toggle_night(True)
                     # Nights last for between 5 and 25 turns.
```

Comment:
Sorry to nitpick, but you'll also need to update the comment here.

Reply:
Of course, thanks for spotting that one!